rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/LRARecord.java

```diff
@@ -718,8 +745,14 @@ boolean forget() {
                     forgetURI, e.getMessage());
                 // TODO write a test to ensure that recovery only retries the forget request
                 return false; // force recovery to keep retrying
+            } catch (Exception e) {
+                LRALogger.logger.infof("LRARecord.forget at '%s' could not reach participant: %s",
```

Comment:
I would use debug here (failure to contact a participant is not particularly noteworthy since it is often an expected outcome in a distributed system).

Reply:
ok